start

```diff
@@ -30,6 +31,11 @@ def translation_volume_mounts() -> Iterator[str]:
 
 
 def main() -> None:
+    print(
+        "Warning: this may be using an outdated version of the app. Run "
```

Comment:
Nit:
Do we want to match the two warnings (the second one is all upper-case letters)?
```
Warning: this may be using an outdated version of the app. Run `docker pull qiskit/documentation` to check for updates.
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested 
```

Reply:
Only people using Apple Silicon will see the second message, so I don't think it matters.